server/src/middlewares/timeline.test.js

```diff
@@ -0,0 +1,66 @@
+import { verifyTimelineId, verifyTimelineObject } from './timeline';
+
+describe('middleware: timeline', () => {
+  describe('verifyTimelineId: req handler create', () => {
+    let mw;
+    beforeAll(() => {
+      mw = verifyTimelineId;
+    });
+    it('should return a function', () => {
```

Comment:
You have switched the bodies of this and the test below it

Reply:
changed in latest commit